### ISSUE TITLE
C398355 - fails to import symbol

### ISSUE THREAD

**Issue by bzed** [uPesy/easyeda2kicad.py#69]
C398355 is a dual opamp, its symbol on easyeada comes in two(?) parts, the import basically imports an empty symbol.

There is a version of the chip in kicad, there you have to add three symbols and end up with more than one footpriunt that needs to be aligned. Both versions are not nice :)

**Comment by uPesy** [comment]
Yeah, i don't think that the current version of easyeda2kicad support to convert a symbol that is split in severals parts. I have not manage this case :/

**Comment by bzed** [comment]
Stale, but I think worth fixing it.

**Comment by uPesy** [comment]
I don't have the time to implement this right now sorry

**Comment by fpovoski** [comment]
Suddenly(as in this morning), after importing numerous symbols into easyeda2kicad.kicad_sym, any further imports the program completes without errors, yet no symbol is actually populated in the kicad_sym file. Any ideas why?

**Comment by uPesy** [comment]
Fix in #184
